# zero pose (starting pose) of arm and hand to use when simulating
zeros:
  shoulder_pan_joint: -1.085
  shoulder_lift_joint: -1.158
  elbow_joint: 1.924
  wrist_1_joint: 3.868
  wrist_2_joint: -1.0
  wrist_3_joint: -0.5

  joint_0: 0
  joint_1: 0
  joint_2: 1.1
  joint_3: 0

  joint_4: 0
  joint_5: 0
  joint_6: 1.1
  joint_7: 0

  joint_8: 0
  joint_9: 0
  joint_10: 1.1
  joint_11: 0

  joint_12: 0.6
  joint_13: 1.1
  joint_14: 0
  joint_15: 1.1
